Jellyfin.Api/Controllers/NotificationsController.cs

```diff
@@ -0,0 +1,137 @@
+        /// <returns>Notifications summary for the user.</returns>
+        [HttpGet("{UserID}/Summary")]
+        public NotificationsSummaryDto GetNotificationsSummary(
+            [FromRoute] string userId)
+        {
+            return new NotificationsSummaryDto();
+        }
+
+        /// <summary>
+        /// Endpoint for getting notification types.
+        /// </summary>
+        /// <returns>All notification types.</returns>
+        [HttpGet("Types")]
+        public IEnumerable<NotificationTypeInfo> GetNotificationTypes()
+        {
+            return _notificationManager.GetNotificationTypes();
+        }
+
+        /// <summary>
+        /// Endpoint for getting notification services.
+        /// </summary>
+        /// <returns>All notification services.</returns>
+        [HttpGet("Services")]
+        public IEnumerable<NameIdPair> GetNotificationServices()
+        {
+            return _notificationManager.GetNotificationServices();
+        }
+
+        /// <summary>
+        /// Endpoint to send a notification to all admins.
+        /// </summary>
+        /// <param name="name">The name of the notification.</param>
+        /// <param name="description">The description of the notification.</param>
+        /// <param name="url">The URL of the notification.</param>
+        /// <param name="level">The level of the notification.</param>
+        [HttpPost("Admin")]
+        public void CreateAdminNotification(
+            [FromForm] string name,
+            [FromForm] string description,
+            [FromForm] string? url,
+            [FromForm] NotificationLevel? level)
+        {
+            var notification = new NotificationRequest
+            {
+                Name = name,
+                Description = description,
+                Url = url,
+                Level = level ?? NotificationLevel.Normal,
+                UserIds = _userManager.Users.Where(i => i.Policy.IsAdministrator).Select(i => i.Id).ToArray(),
+                Date = DateTime.UtcNow,
+            };
+
+            _notificationManager.SendNotification(notification, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Endpoint to set notifications as read.
+        /// </summary>
+        /// <param name="userId">The userID.</param>
+        /// <param name="ids">The IDs of notifications which should be set as read.</param>
+        [HttpPost("{UserID}/Read")]
+        public void SetRead(
+            [FromRoute] string userId,
+            [FromForm] List<string> ids)
+        {
+        }
+
+        /// <summary>
+        /// Endpoint to set notifications as unread.
+        /// </summary>
+        /// <param name="userId">The userID.</param>
+        /// <param name="ids">The IDs of notifications which should be set as unread.</param>
+        [HttpPost("{UserID}/Unread")]
+        public void SetUnread(
+            [FromRoute] string userId,
+            [FromForm] List<string> ids)
```

Comment:
The existing endpoint has this as a `query` parameter, and is a `string`, not a `List<string>`